validator/boolean.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/googlegenomics/ga4gh-identity"
+)
+
+// Or is a ga4gh.Validator that succeeds iff one of the Validators it wraps
+// returns true.
+type Or []ga4gh.Validator
+
+// Validate returns true iff one of the validators that this Or wraps returns
+// true.  If any of the validators return an error then an error is returned.
+func (or Or) Validate(ctx context.Context, identity *ga4gh.Identity) (bool, error) {
+	results := make([]bool, 0, len(or))
+	for _, v := range or {
+		ok, err := v.Validate(ctx, identity)
+		if err != nil {
+			return false, fmt.Errorf("nested validator: %v", err)
```

Comment:
I think it will be hard to track down problems unless you include some indication of what validator failed